plugins/git4idea/resources/messages/GitBundle.properties

```diff
@@ -195,7 +195,9 @@ rebase.option.onto=Specify a new base for the rebased commits
 rebase.option.interactive=Edit commits before rebasing
 rebase.option.rebase.merges=Recreate commits topology
 rebase.option.switch.to.branch=Select another branch to rebase
-rebase.option.keep.empty=Do not remove empty commits during rebase
+rebase.option.keep.empty=Do not remove initially-empty commits during rebase
+rebase.option.empty.keep=Do not remove commits that become empty during rebase
+rebase.option.empty.drop=Remove commits that become empty during rebase
```

Comment:
I suppose `ask` is deliberately skipped since it'd require input on stdin from user, right?

Reply:
Asking what to do with commits that become empty is what couses this problem in the first place (interactive mode has `--empty=ask` by default), since git4idea does not handle it correctly. But sure, it should be there for completeness sake I will open an issue since it's a bigger fix.